http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockSamDAO.scala

```diff
@@ -50,6 +50,8 @@ class MockSamDAO extends SamDAO[IO] {
           .map(_.contains(action)) //open the option to peek the set: Option[Bool]
           .getOrElse(false) //unpack the resulting option and handle the disk never having existed
         IO.pure(res)
+      case SamResourceType.App =>
```

Comment:
@jdcanas   just fyi,  I'm fixing this in this PR...it's a bit distracting seeing 3 compilation errors everytime